Hospital.Api.Host/Controllers/AnalyticsController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Hospital.Application.Contracts;
+using Hospital.Application.Contracts.Appointments;
+using Hospital.Application.Contracts.Doctors;
+using Hospital.Application.Contracts.Patients;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Hospital.Api.Host.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AnalyticsController(IAnalyticsService service) : ControllerBase
+{
+    [HttpGet("doctors/10plus")]
+    public ActionResult<IEnumerable<DoctorDto>> Doctors10Plus()
+        => Ok(service.GetDoctorsWith10Plus());
```

Comment:
Вы оперируете наборами данных. Возможна ситуация когда у вас потеряется доступ к данным или во время работы возникнет еще какая то ошибка. 
Хотелось бы чтобы методы контроллеров содержали хотя бы примитивную обработку ошибок. 
Ошибку стоит залогировать и отобразить пользователю сообщение об ошибке (желательно в усеченном виде не раскрывая важной информации которая может содержаться в ex.Message).
```suggestion
    public ActionResult<IEnumerable<DoctorDto>> Doctors10Plus()
    {
        try
        {
            return Ok(service.GetDoctorsWith10Plus());
        }
        catch (Exception ex)
        {
            logger.LogError(ex, "Error retrieving doctors with 10+ years experience");
            return BadRequest("An error occurred while processing the request to get doctors with 10+ years of experience");
        }
    }
```
Добавление обработчика ошибок является признаком хорошего тона в программировании, так мы фиксируем у себя ошибку и не даем лишней информации пользователям.
В ASP.NET Core уже есть встроенный логгер, вы можете использовать его получив его через DI. Для этого в конструкторе укажите `ILogger<AnalyticsController> logger`